backend/Dockerfile

```diff
@@ -15,6 +15,8 @@ RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
     --mount=type=cache,target=/var/lib/apt,sharing=locked \
     apt-get update && apt-get install -y libssl-dev openssl curl pkg-config
 
+RUN cargo install sqlx-cli --no-default-features --features native-tls,postgres
```

Comment:
note - the sqlx project doesn't distribute a binary so I need to compile it for use in the init contianer